doc/source/users/scan.rst

```diff
@@ -220,6 +220,22 @@ Following variables are supported:
     Its value is a dictionary with keys pointing to the server side directory
     and values to the client side directory/ies (string or list of strings).
 
+**OutputBlock**
+    Enable/disable printing the scan output records at the same line instead
+    of appending records as new lines a.k.a. block mode. This can save space
+    of the console output and facilitate an easy visual access to the scan
+    header printed at the beginning of the scan - especially usefull for very
+    long scans.
+
+**OutputCols**
+    Select a subset of the scan output columns to be printed. Two different
+    ways of specifying the selected colums exists. First, by specifying an
+    integer number e.g. 2 means print just the very first two colums.
+    Second, by specifying a list of string identifiers e.g. scan elements URIs
+    or the scan record metadata identifiers: "point_nb", "timestamp".
```

Comment:
It could be usefull to place example of setting OutputCols e.g. 
`senv OutputCols "['tango_host:port/expchan/zerodctrl01/1','motor1','timestamp']"`
It took me quite long time to discover proper syntax.

Reply:
Very good point! I was also thinking about it. Actually what I wrote in the documentation, about the scan elements URIs, is not 100% right at the moment but what we aim to. This is exactly the same as explained in: #500. I hope that we will get to a unified references ASAP and what we have now is just a buggy behavior.

Then, the `senv` macro syntax e.g. how to pass a list of strings in spock, could be explained in the senv macro documentation.